tasks/mailcowconf.yml

```diff
@@ -98,6 +98,13 @@
     replace: "SOLR_HEAP={{ mailcow__config_solr_heap }}"
   notify: Recreate mailcow
 
+- name: Configure SKIP_SOLR
```

Comment:
Can you please update the task name to reflect the config option you are changing :)

Reply:
Sorry, I forgot. Of course I changed it